sections/semantics-document-metadata.include

```diff
@@ -290,18 +290,18 @@
   <div class="example">
     In this example, a <{base}> element is used to set the <a>document base URL</a>:
 
-    <pre highlight="html">
-      &lt;!DOCTYPE html>
-      &lt;html lang="en">
-        &lt;head>
-          &lt;title>This is an example for the &amp;lt;base&amp;gt; element&lt;/title>
-          &lt;base href="https://www.example.com/news/index.html">
-        &lt;/head>
-        &lt;body>
-          &lt;p>Visit the &lt;a href="archives.html">archives&lt;/a>.&lt;/p>
-        &lt;/body>
-      &lt;/html>
-    </pre>
+    <xmp highlight="html">
+      <!DOCTYPE html>
+      <html lang="en">
+        <head>
+          <title>This is an example for the &lt;base&gt; element</title>
```

Comment:
stray \&lt;

Reply:
thank you!

Reply:
These aren't stray. They're needed to display the `<base>` as is.

Reply:
Actually you don't within a `title` element. It [uses the RCDATA parsing](http://w3c.github.io/html/syntax.html#the-in-head-insertion-mode) path so the [text goes into the DOM](http://software.hixie.ch/utilities/js/live-dom-viewer/?<!DOCTYPE%20html>%0A<title>add%20<mark>%20in%20title<%2Ftitle>%0A<p>add%20<mark>%20in%20p%0A<xmp>add%20<mark>%20in%20xmp<%2Fxmp>%0A<textarea>add%20<mark>%20in%20textarea<%2Ftextarea>%0A<pre>add%20<mark>%20in%20pre<%2Fpre>)

I am not sure if I would recommend writing HTML this way, though. Question for the Style guide #1193 I suspect